tool/tctl/common/auth_command.go

```diff
@@ -363,10 +363,7 @@ func (a *AuthCommand) generateSnowflakeKey(ctx context.Context, clusterAPI auth.
 // RotateCertAuthority starts or restarts certificate authority rotation process
 func (a *AuthCommand) RotateCertAuthority(ctx context.Context, client auth.ClientI) error {
 	if a.rotateType == "" {
-		return trace.BadParameter("required flag --type not provided; previous versions defaulted to --type=all which is deprecated and will be removed in a future version")
-	}
-	if a.rotateType == string(types.CertAuthTypeAll) {
-		fmt.Println("\033[0;31mNOTICE:\033[0m --type=all will be deprecated in a future version")
+		return trace.BadParameter("required flag --type not provided")
```

Comment:
Why not just mark the flag required with kingpin and let kingpin handle this check?